text/0000-shared-layers.md

```diff
@@ -0,0 +1,117 @@
+# Meta
+[meta]: #meta
+- Name: Shared cached build layers
+- Start Date: 2021-05-21
+- Author(s): [@samj1912](https://github.com/samj1912)
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: (put "N/A" unless this replaces an existing RFC, then link to that RFC)
+
+# Summary
+[summary]: #summary
+
+This RFC introduces the concept of shared build layers which multiple buildpacks can collaborate on/re-use during build time which are not meant to end up in the final application image and are cacheable. Typical use cases include shared caches for tools like CCache, pip cache etc.
+
+> NOTE: This RFC should be implemented as an atomic change alongside RFC https://github.com/buildpacks/rfcs/pull/155
```

Comment:
```suggestion
> NOTE: This RFC should be implemented as an atomic change alongside [RFC#155](https://github.com/buildpacks/rfcs/pull/155)
```

Reply:
@hone is that notation confusing given the RFC number doesn't match the PR number. E.g. if the referenced PR were merged today it would be RFC 0089.

Reply:
Yeah, it might be but we seem to use "RFC#XXX" for GitHub PR numbers and "RFC XXXX" for RFC numbers.